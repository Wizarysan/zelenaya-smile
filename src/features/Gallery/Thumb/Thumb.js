import React from 'react';
import {Link} from 'react-router';

class Thumb extends React.Component {
  render() {
    return (
      <div className="gallery__body__thumb">
        <Link to={'/image/'+this.props.currentId+'/'+this.props.image+'/'+this.props.currentCity}>
          <img className="gallery__body__thumb-img" src={process.env.PUBLIC_URL+"/galleries/"+this.props.currentId+"/small/"+this.props.image} />
        </Link>
      </div>
    );
  }
}

export default Thumb;
