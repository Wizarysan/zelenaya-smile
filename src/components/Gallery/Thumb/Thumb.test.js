import React from 'react';
import Thumb from './Thumb.js';
import renderer from 'react-test-renderer';

it('renders correctly with test ID, city and image', () => {
  const tree = renderer
    .create(<Thumb currentId="1" currentCity="testCity" image="test.jpg"/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});