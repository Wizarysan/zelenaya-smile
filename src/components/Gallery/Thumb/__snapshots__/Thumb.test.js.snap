exports[`test renders correctly with test ID, city and image 1`] = `
<div
  className="gallery__body__thumb">
  <a
    onClick={[Function]}
    style={Object {}}>
    <img
      alt=""
      className="gallery__body__thumb-img"
      src="/galleries/1/small/test.jpg" />
  </a>
</div>
`;
